test/runtime_tests.cc

```diff
@@ -1013,5 +1015,50 @@ namespace detail {
 		dry_run_with_nodes(nodes);
 	}
 
+	TEST_CASE_METHOD(test_utils::mpi_fixture, "Config reads correctly environment varibles", "[env-vars]") {
+		std::unordered_map<std::string, std::string> valid_test_env_vars{//
+		    {"CELERITY_LOG_LEVEL", "debug"},                             //
+		    {"CELERITY_GRAPH_PRINT_MAX_VERTS", "1"},                     //
+		    {"CELERITY_DEVICES", "0 0"},                                 //
+		    {"CELERITY_PROFILE_KERNEL", "1"},                            //
+		    {"CELERITY_DRY_RUN_NODES", "4"}};
```

Comment:
Adding a additional comma after the last entry (`{"CELERITY_DRY_RUN_NODES", "4"},`) will tell `clang-format` to format it like this, without the need to force this layout by adding empty comments.